account_invoice_mgmt_dms/models/account_move.py

```diff
@@ -58,6 +71,14 @@ def _compute_complete_proceesing_id(self):
             if purchase_invoice_file:
                 record.purchase_invoice_proceesing_id = purchase_invoice_file
 
+    def action_post(self):
+        if self.type in ['in_invoice', 'in_refund', 'in_receipt']:
+            if self.partner_id.validator_complete_proceesing_id and not self.validator_complete_proceesing_id:
+                self.validator_complete_proceesing_id = self.partner_id.validator_complete_proceesing_id
+            if not self.validator_complete_proceesing_id:
+                raise ValidationError(_("You must select a validator"))
```

Comment:
`action_post` is multi, see https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/addons/account/models/account_move.py#L2443 and https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/addons/account/models/account_move.py#L2334

Then, code should consider this and add invoice number in message error